app/helpers/urls.js

```diff
@@ -21,6 +21,12 @@ const getUrl = (urlObject, url, request, secBtn, _currentUrl) => {
     return  dependentAnswer != null ? thenUrl : elseUrl
   }
 
+  if (elseUrl === 'replacing-insulation' && 
```

Comment:
not sure why this if statement is needed. Is there no way to use nonDependentUrl for this url switching instead?